From 6d3b78207a45077e496bed4f7b43b93d41470480 Mon Sep 17 00:00:00 2001
From: Jakub Ruzicka <jruzicka@redhat.com>
Date: Mon, 16 Jun 2014 16:04:11 +0200
Subject: [PATCH] Remove runtime dependency on python-pbr

---
 setup.py                |  1 -
 troveclient/__init__.py | 10 +---------
 2 files changed, 1 insertion(+), 10 deletions(-)

diff --git a/setup.py b/setup.py
index 7363757..09230a0 100755
--- a/setup.py
+++ b/setup.py
@@ -26,5 +26,4 @@ except ImportError:
     pass
 
 setuptools.setup(
-    setup_requires=['pbr'],
     pbr=True)
diff --git a/troveclient/__init__.py b/troveclient/__init__.py
index a470d7b..65c0bac 100644
--- a/troveclient/__init__.py
+++ b/troveclient/__init__.py
@@ -16,12 +16,4 @@
 
 __all__ = ['__version__']
 
-import pbr.version
-
-version_info = pbr.version.VersionInfo('python-troveclient')
-# We have a circular import problem when we first run python setup.py sdist
-# It's harmless, so deflect it.
-try:
-    __version__ = version_info.version_string()
-except AttributeError:
-    __version__ = None
+__version__ = "REDHATTROVECLIENTVERSION"
