diff -up troveclient/__init__.py.bak troveclient/__init__.py
--- troveclient/__init__.py.bak	2014-05-08 11:31:24.582007093 +0200
+++ troveclient/__init__.py	2014-05-08 11:34:21.633615528 +0200
@@ -16,14 +16,4 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-__all__ = ['__version__']
-
-import pbr.version
-
-version_info = pbr.version.VersionInfo('python-troveclient')
-# We have a circular import problem when we first run python setup.py sdist
-# It's harmless, so deflect it.
-try:
-    __version__ = version_info.version_string()
-except AttributeError:
-    __version__ = None
+__version__ = "RPMVERSION"
